### PR TITLE
Instructions on how to modify the worker pool after an image has been marked deprecated

### PR DESCRIPTION
**Pull request by benPearce1**

### CODE REVIEW

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 1]
```diff
@@ -62,6 +73,7 @@ Each `Windows Server Core 2019` worker is provisioned with a baseline of tools i
 - Pip (20.1.1)
 - Powershell Core (latest)
 - Python (3.7.4)
+- GCloud CLI (339.0.0)
```
Comment (benPearce1) [inline docs/infrastructure/workers/dynamic-worker-pools.md:76]: Small note missed in a previous piece of work

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 2]
```diff
@@ -100,6 +113,7 @@ Each `Ubuntu Server 18.04` worker is provisioned with a baseline of tools includ
 - Docker (latest)
 - Powershell Core (latest)
 - Python 3 (latest)
+- GCloud CLI (339.0.0)
```
Comment (benPearce1) [inline docs/infrastructure/workers/dynamic-worker-pools.md:116]: Small note missed in a previous piece of work

File: docs/infrastructure/workers/dynamic-worker-pools.md [hunk 3]
```diff
@@ -87,6 +99,7 @@ Each `Windows Server Core 2016` worker is provisioned with a baseline of tools i
 - Pip (20.1.1)
 - Powershell Core (latest)
 - Python (3.7.4)
+- GCloud CLI (339.0.0)
```
Comment (benPearce1) [inline docs/infrastructure/workers/dynamic-worker-pools.md:102]: Small note missed in a previous piece of work

**Review by pawelpabich — APPROVED** [review]
👍

**Pull request merged**